modeling-acceptance-tests/pom.xml

```diff
@@ -11,10 +11,29 @@
   <artifactId>modeling-acceptance-tests</artifactId>
 
   <dependencies>
+    <dependency>
+      <groupId>org.activiti.cloud</groupId>
+      <artifactId>activiti-cloud-services-org-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
@constantin-ciobotaru this is something that I don't like here.. The main idea is that acceptance test should only have API types and probably the marshallers if needed. In the runtime side we have a model-api (only interfaces, not jackson stuff) module and a model-impl module which contains the spring and jackson deps. Can we follow the same approach here. It doesn't make any sense to me to have the JPA dependency in the client side.